src/wazuh_db/wdb.c

```diff
@@ -118,8 +118,10 @@ static const char *SQL_STMT[] = {
     [WDB_STMT_FIM_DELETE_AROUND] = "DELETE FROM fim_entry WHERE file < ? OR file > ?;",
     [WDB_STMT_FIM_DELETE_RANGE] = "DELETE FROM fim_entry WHERE file > ? AND file < ?;",
     [WDB_STMT_FIM_CLEAR] = "DELETE FROM fim_entry;",
+    [WDB_STMT_SYNC_UPDATE_ATTEMPT_LEGACY] = "UPDATE sync_info SET last_attempt = ? WHERE component = ?;",
```

Comment:
In fact, the `n_attempts` variable will be updated in the `WDB_STMT_SYNC_UPDATE_COMPLETION` query, so it won't be 0.
This extra query was added to avoid the overflow of this value in legacy agents, because it's called for every package and not for every synchronization cycle.